packages/bentley/grouping-mapping-widget/public/locales/en/GroupingMappingWidget.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Any reason you need to add this even though it's empty? If you don't register it, apps will have to make one less network call when using the component.